lib/Models/CatalogItem.js

```diff
@@ -163,6 +163,12 @@ var CatalogItem = function(terria) {
      */
     this.nowViewingMessage = undefined;
 
+    /**
+     * Gets or sets a template to display message in a info box.
+     * @type {String} Html template that can be used to display info
+     */
+    this.infoTemplate = undefined;
```

Comment:
Mind renaming this to `featureInfoTemplate`?  I should have realized it sooner, but `infoTemplate` is a big ambiguous considering that `CatalogItem` has an `info` property that is completely separate from this.